src/scripts/interaction.js

```diff
@@ -1598,6 +1610,16 @@ function Interaction(parameters, player, previousState) {
         });
 
         instance.on('click', function (event) {
+
+          if(instance.libraryInfo.machineName === "H5P.IVHotspot" && parameters.wrongAnswerParams.isWrongChoice) {
```

Comment:
Formatting: space between if and (
Do you have auto-formatting set up?

Reply:
Fixed
Actually I have, I'll check If it is configured